### PR TITLE
Publishing failover script by Ops

### PR DESCRIPTION
**Pull request by JermilaPaulDhas**
ops have a set of scripts that they use for publishing failover cluster. 
Integrating this with the provisioner

### CODE REVIEW

**Comment by JermilaPaulDhas** [comment]
[Update the new jumpbox with Ops publishing failover docs](https://trello.com/c/3p9qFDKU/667-update-the-new-jumpbox-with-ops-publishing-failover-docs)

File: content-jumpbox-provisioner/files/pre_checks.sh [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  echo $warning_pods
+  echo $completed_pods
+  echo "Other pod status = `kubectl get pods | egrep -iv "running|warning|completed" | awk '{if(NR>1)print}' | wc -l`"
+  echo ""
+  echo "###########################"
+  echo ""
+  echo ""
+  if [ "$other_pods" = "0" ]
+    then
+      echo "_/﹋\_"
```
Comment (tamas-molnar) [inline content-jumpbox-provisioner/files/pre_checks.sh]: Why is the same ascii art person saying every message?

File: content-jumpbox-provisioner/files/pre_checks.sh [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+  header_func
+  echo "Command will execute at $i"
+  echo ""
+  echo ""
+  sleep 2
+  clear
+  done
+  echo ""
+  echo ""
+  echo "       \$\$\$\$"
```
Comment (tamas-molnar) [inline content-jumpbox-provisioner/files/pre_checks.sh]: what is this supposed to be? :))

**Comment by JermilaPaulDhas** [comment]
@tamas-molnar these are this scripts that DutyOps use to failback publishing when it fails to EU due to kafka blips. It checks if the publishing cluster in EU is healthy and then fails back and waits till dyn switch has happened.

I didn't write the script so I can't tell you why we have those characters. But what I can tell you is they look cool on the terminal.

DutyOps currently use these script to failback publishing. I thought the jumpbox would be a good place to put it in coz this server is meant for them to use

**Pull request merged**